shell.nix

```diff
@@ -114,41 +82,12 @@ let
       ${localLib.optionalString pkgs.stdenv.isLinux "export XDG_DATA_HOME=$HOME/.local/share"}
       cp -f ${daedalusPkgs.iconPath.${cluster}.small} $DAEDALUS_INSTALL_DIRECTORY/icon.png
 
-      # These links will only occur to binaries that exist for the 
+      # These links will only occur to binaries that exist for the
       # specific build config
       ln -svf $(type -P jormungandr)
       ln -svf $(type -P cardano-wallet-jormungandr)
-      ln -svf $(type -P cardano-http-bridge)
-      ln -svf $(type -P cardano-wallet-http-bridge)
       ln -svf $(type -P jcli)
 
-      ${pkgs.lib.optionalString autoStartBackend ''
-        for x in wallet-topology.yaml log-config-prod.yaml configuration.yaml mainnet-genesis-dryrun-with-stakeholders.json ; do
-          ln -svf ${daedalusPkgs.daedalus.cfg}/etc/$x
-        done
-        STATE_PATH=$(eval echo $(jq ".statePath" < ${launcher-json}))
-        ${pkgs.lib.optionalString (cluster == "demo") ''
-          ln -svf ${demoTopologyYaml} wallet-topology.yaml
-
-          # Refresh the old state directory
-          rm -rf "''${STATE_PATH}"
```

Comment:
When developing out of nix-shell, I think it will remain desirable to rm -rf & mkdir the `STATE_PATH`. Due to how often formats are currently changing, if we don't do this, then its possible a developer will pull the most recent HEAD, do `yarn nix:dev` -> `yarn dev` and the node won't start due to out-of-date config files or incompatible sqlite state for jormungandr